client.go

```diff
@@ -14,7 +14,10 @@ import (
 )
 
 type Result struct {
-	Latency float64 `json:"latency"`
+	Type          string  `json:"type"`
+	TimeSeconds   float64 `json:"timeSeconds"`
+	UploadBytes   uint    `json:"uploadBytes"`
+	DownloadBytes uint    `json:"downloadBytes"`
```

Comment:
It's better to use uint64 here, otherwise we'll quickly run into problems on 32 bit systems.